distributed/nanny.py

```diff
@@ -568,6 +571,8 @@ async def close(self, comm=None, timeout=5, report=None):
 
         self.status = Status.closing
         logger.info("Closing Nanny at %r", self.address)
+        if not report:
+            logger.info("Not reporting closure to scheduler")
```

Comment:
```suggestion
        logger.info(f"Closing Nanny at {self.address!r}. Report closure to scheduler: {report}")
```
Single log line is nicer, plus it might be just as useful to log that the report _should_ happen.